src/index.html

```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="search search-bar__big"
+      method="get"
+      action="https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md"
+      data-qa="big"
+    >
```

Comment:
Fix code style everywhere
```suggestion
    <form
      class="search search-bar__big"
      method="get"
      action="#"
      data-qa="big"
    >
```